### DISCUSSION TITLE
Layer Cake in the wild: Live examples from across the web

### DISCUSSION THREAD

**Discussion by mhkeller** [mhkeller/layercake#168] [Show and tell]
Here's a list of various examples of Layer Cake being used out there in the world. Feel free to add your own! I'll add to this over time.

1. [The Star Tribune – Which Minnesota schools are doing better than expected?](https://www.startribune.com/which-minnesota-schools-are-doing-better-than-expected-poverty-levels-math-reading-scores-mcas/600307963/)
2. [The Star Tribune – Peak pickle? 10 charts show Minnesota State Fair food trends, present and past](https://www.startribune.com/peak-pickle-10-charts-show-minnesota-state-fair-food-trends-present-and-past/600298849/)
3. [The Star Tribune – Driest spring in 5 decades leaves Minnesota farmers with withered crops](https://www.startribune.com/its-rained-less-in-the-twin-cities-since-may-than-any-year-for-the-past-5-decades/600284528/)
4. [The Star Tribune – What's passed and what's still to come at the Minnesota Legislature this year](https://www.startribune.com/recreational-marijuana-social-security-tax-relief-track-bills-minnesota-legislature/600259167/)
5. [The Star Tribune – Minnesota's high school graduation rates rebounded in 2022. Look up your school's results](https://www.startribune.com/2022-minnesota-high-school-graduation-rates-rebound-to-pre-pandemic-levels-look-up-your-results/600269652/)
6. [The Star Tribune – See where your city ranks in the Twin Cities area's hottest housing markets](https://www.startribune.com/minneapolis-suburbs-st-paul-twin-cities-hot-housing-market-sell-buy-where-2023/600246718/)
7. [The Straits Times – Singapore is getting hotter](https://www.straitstimes.com/multimedia/graphics/2023/03/singapore-getting-hotter-temperature-data/index.html)
8. [Urban Institute – Getting a Good Job Depends More on Race and Gender than Education](https://apps.urban.org/features/job-quality-segregation-race-gender/)
9. [Urban Institute – How the Federal Income Tax System Can Worsen Racial Disparities](https://apps.urban.org/features/federal-income-tax-system-can-worsen-racial-disparities/)
10. [USA Facts – Nearly half of American households have no retirement savings](https://usafacts.org/data-projects/retirement-savings)
11. [USA Facts – How much do Americans spend on housing?](https://usafacts.org/data-projects/housing-costs)
12. [Axios - Mainstream media bloodbath: News outlets slash jobs as business suffers](https://www.axios.com/2024/01/26/media-layoffs-strikes-journalism-dying)
13. [Axios - Why new home sales soared and existing homes plunged in 2023](https://www.axios.com/2024/01/26/housing-home-sales-interest-rates)
14. [CSS Labs - YouTube Politics](https://watts-lab.github.io/LivingJournal-YoutubeDashboard/)
15. [NY Times - Where Electric Vehicles Are (and Aren’t) Taking Off Across the U.S.](https://www.nytimes.com/interactive/2024/03/06/climate/hybrid-electric-vehicle-popular.html)
16. [NY Times - Prescriptions Surged as Trump Praised Drugs in Coronavirus Fight](https://www.nytimes.com/2020/04/25/us/coronavirus-trump-chloroquine-hydroxychloroquine.html)
17. [NY Times - New SAT Data Highlights the Deep Inequality at the Heart of American Education](https://www.nytimes.com/interactive/2023/10/23/upshot/sat-inequality.html)
18. [NY Times - Explore How Income Influences Attendance at 139 Top Colleges](https://www.nytimes.com/interactive/2023/10/23/upshot/sat-inequality.html)
19. [NY Times - Why School Absences Have ‘Exploded’ Almost Everywhere](https://www.nytimes.com/interactive/2024/03/29/us/chronic-absences.html)
20. [NY Times - ‘Monster Fracks’ Are Getting Far Bigger. And Far Thirstier](https://www.nytimes.com/interactive/2023/09/25/climate/fracking-oil-gas-wells-water.html)
21. [Economist - Sun Machines](https://www.economist.com/interactive/essay/2024/06/20/solar-power-is-going-to-be-huge)

**Comment by rgieseke** [comment]
It's only a tiny graph but it was nice to easily be able to get interaction between the Leaflet map and the Layer Cake chart.

https://climatevulnerabilitymonitor.org/biophysical/deu/temperature/mean-surface-temperature/

**Reply by mhkeller** [reply]
Ah neat that's really cool!

**Comment by benkates** [comment]
Thanks for adding some of our work from the Urban Institute! We'll add as new projects get released. I know the Axios graphics team uses Layer Cake, [recent work here](https://www.axios.com/bitcoin-etf-approval-sec-crypto-saga). Cheers.

**Reply by mhkeller** [reply]
Wow what a scroll! Cool presentation.

**Comment by rachelmarconi** [comment]
From the Urban Institute this month! lines (but a custom lollipop)
[https://apps.urban.org/features/medical-debt-over-time/](https://apps.urban.org/features/medical-debt-over-time/)

**Comment by mhkeller** [comment]
Here's one from Germany. Really neat animated maps and also some beeswarm to bar-chart transitions: https://interaktiv.tagesspiegel.de/waffen/europaeische-waffen-amerikanische-opfer/
